docs/14_0_0/gettingstarted/community.md

```diff
@@ -0,0 +1,11 @@
+# PrimeFaces Community
+
+PrimeFaces has a vibrant community that enhances its functionality. The list below is by no means comprehensive.
+
+## Community Projects
+
+* [PrimeFaces Extensions](https://primefaces-extensions.github.io) includes many more components such as `GChart` and `ImageZoom`
+* [Arquillian Primefaces](https://github.com/omnifaces/arquillian-primefaces) easier integration testing of PrimeFaces components
+* [FlowLogix JPALazyDataModel](https://docs.flowlogix.com/#section-jpa-lazymodel)
+  fully integrates with Jakarta EE and `@Inject`able
+* [OptimusFaces](https://github.com/omnifaces/optimusfaces) Service-based implementation of `JPALazyDataModel`
```

Comment:
Can you add `Quarkus Faces` here: https://github.com/melloware/quarkus-faces

Reply:
done